Assets/Mirage/Runtime/NetworkSceneManager.cs

```diff
@@ -358,18 +362,30 @@ private UniTask ServerSceneLoading(string scenePath, IEnumerable<INetworkPlayer>
 
             SetAllClientsNotReady(players);
 
-            OnServerStartedSceneChange?.Invoke(scenePath, sceneOperation);
 
-            if (Server.LocalClientActive)
+            bool serverSceneIsLoaded = GetSceneByPathOrName(scenePath).isLoaded;
+
+            //Only do a guarantee load if we are loading multiple additive scenes, otherwise only load if unloaded
+            bool serverShouldLoad = loadMultipleAdditiveScenes ? true : !serverSceneIsLoaded;
+
+            if (serverShouldLoad)
```

Comment:
3 of these `if` blocks check `if (serverShouldLoad)` they should probably just be 1 block instead